docs/fsharp/get-started/get-started-visual-studio.md

```diff
@@ -24,14 +24,15 @@ You'll next see a list of Workloads. You can install F# through any of the follo
 
 |Workload|Action|
 |--------|------|
-|.NET desktop development|Select **F# language support** from the right-hand side|
-|ASP.NET and web development|Select **F# language support** from the right-hand side|
-|Data storage and processing|Select **F# language support** from the right-hand side|
-|Mobile development with .NET|No action - F# is installed by default|
+| .NET Core cross-platform development | No action - F# is installed by default |
+| ASP.NET and web development | No action - F# is installed by default |
+| Azure development | No action - F# is installed by default |
+| Mobile development with .NET |No action - F# is installed by default|
+| Data science and analytical applications | None. F# is installed by default |
```

Comment:
it would be good to be consistent:
![image](https://user-images.githubusercontent.com/12971179/32975357-5d2c2338-cbbb-11e7-8c8e-7c290aeebf31.png)

Reply:
Done it!